.github/workflows/build-release.yml

```diff
@@ -57,3 +57,14 @@ jobs:
         path: .
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+    - name: Trigger Docker build workflow
```

Comment:
Thanks for the PR! Quick q: would this only run for releases in our `main` branch?

Reply:
Great! Thanks for the clarification!